nlfyapp/src/features/onboarding.screen.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable prettier/prettier */
+import React from "react";
+import { Text, Image, View, StyleSheet,StatusBar, SafeAreaView, Pressable } from "react-native";
+import Swiper from "react-native-swiper";
+import styled from "styled-components/native";
+
+
+const SafeArea = styled(SafeAreaView)`
+  flex: 1;
+  margin-top: ${StatusBar.currentHeight}px;
+`;
+
+// const wrapper = styled(View)``;
+
+const Slide1 = styled(View)`
+  flex : 1;
+  justify-content : center;
+  align-items : center
+`;
+
+const Slide2 = styled(View)`
+  flex : 1;
+  justify-content : center;
+  align-items : center
+`;
+
+const Slide3 = styled(View)`
+  flex : 1;
+  justify-content : center;
+  align-items : center
+`;
+
+const Slide1Image = styled(Image)`
+  height: 500px;
+  width: 500px;
+  align-self: center;
+  position: absolute;
+  top: 0 
+`;
+
+const Slide2Image = styled(Image)`
+  height: 250px;
+  width: 250px;
+`;
+
+const TextSlide1 = styled(Text)`
+  align-items: center;
+  top: 180px;
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props)=> props.theme.fontSizes.header};
+  font-weight: ${(props)=> props.theme.fontWeights.medium};
+  font-family: ${(props)=> props.theme.fonts.body}; 
+`;
+
+const TextSlide2 = styled(Text)`
+  top: 190px;
+  padding-top: 16px;
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props)=> props.theme.fontSizes.body};
+  font-weight: ${(props)=> props.theme.fontWeights.medium};
+  font-family: ${(props)=> props.theme.fonts.body}; 
+`;
+
+const TextScreen2 = styled(Text)`
+  color: ${(props) => props.theme.colors.text.primary};
+  font-size: ${(props)=> props.theme.fontSizes.body};
+  font-weight: ${(props)=> props.theme.fontWeights.medium};
+  font-family: ${(props)=> props.theme.fonts.body}; 
+`;
+
+const TextScreen2Orange = styled(Text)`
+  top: 20px;
+  padding-top: 16px;
+  color: ${(props) => props.theme.colors.text.secondary};
+  font-size: ${(props)=> props.theme.fontSizes.caption};
+  font-weight: ${(props)=> props.theme.fontWeights.medium};
+  font-family: ${(props)=> props.theme.fonts.body}; 
+`;
+
+const Button = styled(Pressable)`
```

Comment:
Try to create a reusable style for button ( with pressable ) component

Reply:
This will be taken care in the next PR